src/inference/hand_detection.py

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env python3
 
 import cv2
-
-from yolo import YOLO
+import numpy as np
+from inference.yolo import YOLO
 
 
 class HandDetector:
 
-    def __init__(self, image_size=416, confidence=0.2):
-        self.yolo = YOLO("models/cross-hands-tiny.cfg", "models/cross-hands-tiny.weights", ["hand"])
+    def __init__(self, image_size=416, confidence=0.2, model_dirs="models/"):
```

Comment:
models_dir because it's one directory with multiple models, not multiple directories.